rotkehlchen/tests/unit/decoders/test_ens.py

```diff
@@ -726,7 +726,7 @@ def test_set_attribute_for_non_primary_name(database, ethereum_inquirer, ethereu
 
 @pytest.mark.vcr()
 @pytest.mark.parametrize('ethereum_accounts', [['0xF5d90Ac6747CB3352F05BF61f48b991ACeaE28eB']])
-def test_claim_airdrop(database, ethereum_inquirer, ethereum_accounts):
+def test_claim_airdrop(database, ethereum_inquirer, ethereum_accounts, add_subgraph_api_key):  # pylint: disable=unused-argument
```

Comment:
Is it needed for all these tests? Have you re-recorded all these tests in VCR? If it's not needed in a test remove it

Reply:
There were 2 were not. The other requires them